src/modules/endpoint/models/PoiModel.js

```diff
@@ -0,0 +1,92 @@
+// @flow
```

Comment:
Can we maybe create a sub class here? This shares a lot of types with a Page.

Reply:
I think this just adds more complexity, because a page/category has not that many attributes anyway (you would always have to access them by catgory.subclass.shared_type...which would lead to lots of changes). Maybe it makes sense to introduce a custom location type shared by pois and events which contains stuff like town, adress, longitude and latitude and provides some methods for formatting and displaying the address. I would do this in another task though.